docs/docs/cmd/spo/listitem/listitem-roleassignment-add.md

```diff
@@ -0,0 +1,81 @@
+# spo listitem roleassignment add
+
+Adds a role assignment to a listitem.
+
+## Usage
+
+```sh
+m365 spo listitem roleassignment add [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: URL of the site where the listitem is located
+
+`-i, --listId [listId]`
+: ID of the list. Specify either listId, listTitle or listUrl but not multiple.
+
+`-t, --listTitle [listTitle]`
+: Title of the list. Specify either listId, listTitle or listUrl but not multiple.
```

Comment:
Let's remove these short options

Reply:
hmmm but why? should we keep them 🤔 or probably I am missing something 🙂
 I think those are aligned with `listItem list` command 🤔